### DISCUSSION TITLE
Feature Request: Extend Code Generation Feature

### DISCUSSION THREAD

**Discussion by edsheltonorg** [AnWeber/httpyac#489] [General]
Hi, really enjoy the project!

A *similar-in-functionality* extension, REST Client, can generate requests to different languages:

![image](https://github.com/AnWeber/httpyac/assets/87239055/e3d2c162-2661-472e-bf68-876aee4ab45f)

![image](https://github.com/AnWeber/httpyac/assets/87239055/6b39a952-9a29-4ec3-a4c1-f522cbb45e04)

Stepping the extension, it appears to primarily use [Kong's HTTPSnippet](https://github.com/Kong/httpsnippet) library to generate code.

Feel free to ignore this request. Thanks for all the hard work putting this together!

**Comment by AnWeber** [comment]
@edsheltonorg I have integrated the feature in the VSCode extension similar to rest-client. There simply select the HttpRegion and use the command `httpyac.generateCodeSelectLanguage`, or activate the CodeLens for it. I have already considered integrating it into httpyac on the command line as well, however I have lacked the time so far. Also, the current implementation could be tweaked a bit, since the `@ref` feature creates more use cases.

**Reply by edsheltonorg** [reply]
You're 100% right, I don't know how I missed it 😵 

![image](https://github.com/AnWeber/httpyac/assets/87239055/45416f71-9cae-44db-846f-addd4024a10f)

Thanks again for all the hard work on this excellent project!

**Comment by edsheltonorg** [comment]
Turns out dev already had this feature `httpYac: Generate Code with Language Selection`.. woops!
Closing.
